dotcom-rendering/src/types/front.ts

```diff
@@ -134,8 +136,28 @@ export type DCRContainerPalette =
 // TODO: These may need to be declared differently than the front types in the future
 export type DCRContainerType = FEContainerType;
 
-interface FEMediaAtoms {
+/** @see https://github.com/guardian/frontend/blob/0bf69f55a/common/app/model/content/Atom.scala#L191-L196 */
+interface MediaAsset {
+	id: string;
+	version: number;
+	platform: string;
+	mimeType?: string;
+}
+
+/** @see https://github.com/guardian/frontend/blob/0bf69f55a/common/app/model/content/Atom.scala#L158-L169 */
+export interface FEMediaAtoms {
+	id: string;
+	/** @deprecated */
+	defaultHtml: unknown;
+	assets: MediaAsset[];
+	title: string;
 	duration?: number;
+	source?: string;
+	posterImage?: { allImages: Image[] };
+	expired?: boolean;
+	activeVersion?: number;
+	/** @deprecated */
+	channelId?: unknown;
```

Comment:
Why are `defaultHtml` and `channelId` unknown and not string?

Reply:
Based on a [similar question](https://github.com/guardian/dotcom-rendering/pull/8075#discussion_r1247631649) by @jamesgorrie I decided to remove them, but forgot these ones.

My thinking was that we knew these fields come through, but we are not currently using them in any meaningful way. I wanted to make it clear to future travellers. I’ll just comment them out for now.